site/_posts/2016-11-21-manageiq-euwe-ga-announcement.html.md

```diff
@@ -0,0 +1,117 @@
+
+Finally, we noticed that Azure was missing pre- and post-provision methods in automate modeling, so Azure method points have been added for cloud provisioning in Automate.
+
+[![Azure PrePosition](/assets/images/blog/Euwe_GA_azure_preposition.png)](/assets/images/blog/Euwe_GA_azure_preposition.png)
+
+Many corresponding enhancements are applied to Google Compute Engine, including metrics, external proxy support, load balancer refresh, and pre/post-provisioning methods and schema changes in the method class for Automate.
+
+[![Google PostPosition](/assets/images/blog/Euwe_GA_google_postposition_thumb.png)](/assets/images/blog/Euwe_GA_google_postposition.png)
+
+During provisioning, the deprecated images are now hidden. Support for parsing Google health checks is added for load balancer refresh.
+
+In GCE, there are [preemptible instances](https://cloud.google.com/compute/docs/instances/preemptible). With Google’s contributions, we have inventory collection and provisioning of these preemptible VMs. Support for instance retirement is improved too.
+
+## OpenStack Improvements
+
+Working with the OpenStack team, we bring you a long list of improvements for the OpenStack cloud provider. To start off, collection of Host Aggregate inventory allows the OpenStack scheduler to enable advanced scheduling. The mapping of external cloud tenants to ManageIQ tenants is achieved with a post-refresh hook on the OpenStack provider which synchronizes after refresh.
+
+Creating, updating and deleting cloud tenants and host aggregates can now be carried out through the user interface from the Clouds menu under Compute.
+
+[![Add New Cloud Tenant](/assets/images/blog/Euwe_GA_add_cloud_tenant_thumb.png)](/assets/images/blog/Euwe_GA_add_cloud_tenant.png)
+
+Block storage service is introduced with Cinder backup and restore actions. Snapshot functionality has been added for OpenStack cloud volumes and VMs, and you can create and remove snapshots from the UI. New topology view for OpenStack cloud provider shows the availability zones, tenants and instances.
+
+Flavors are mapped to cloud tenants during OpenStack cloud refresh so that they are grouped correctly. Previously Floating IPs can be assigned only during provisioning, however with this release, associating/disassociating Floating IPs can be controlled from the configuration menu after provisioning.
+
+[![Associate Floating IP](/assets/images/blog/Euwe_GA_associate_floating_ip.png)](/assets/images/blog/Euwe_GA_associate_floating_ip.png)
+
+For the OpenStack infrastructure provider, there’s a lot of work done around nodes. The infrastructure nodes can be powered on and off, added or removed. There are additional options for introspecting nodes and setting their provision states. Ironic Controls have been added to the UI, with which Ironic nodes can be registered through Mistral, the OpenStack workflow service. Destroying a node deletes it from Ironic.
```

Comment:
I can't remember the source/sprint I got the info from, but it's [introspecting nodes](http://docs.openstack.org/developer/tripleo-docs/advanced_deployment/introspect_single_node.html)

Thanks for all the comments @mfeifer !!